### ISSUE TITLE
Generic type with `any` in Golang not working

### ISSUE THREAD

**Issue by KiddoV** [gzuidhof/tygo#65]
I have:
```go
//Go
type FeEnum[T any] struct {
	Value  T
	TSName string
}
```

After generate I got:
```ts
//TypeScript
export interface FeEnum<T extends > { //<=== missing `any` here!!!!
  Value: T;
  TSName: string;
}
```

**Comment by spywiree-priv** [comment]
Here is another example:
Go input:
```go
type JsonArray[T any] []T

func (a jsonArray[T]) MarshalJSON() ([]byte, error) {
	if a == nil {
		return []byte("[]"), nil
	}
	return json.MarshalNoEscape([]T(a))
}
```
TS output:
```ts
export type JsonArray = T[];
```
Should be:
```ts
export type JsonArray<T> = T[];
```

**Comment by gzuidhof** [comment]
This should be fixed with #69 in version `0.2.16`.
